### PR TITLE
Zipfldr preparation

### PR DESCRIPTION
**Pull request by learn-more**
Various small commits preparing for zipfldr integration:

- Add com_apitest tests
- Extend CPropertyPageImpl
- Add shell extension debug tool

[CORE-7684](https://jira.reactos.org/browse/CORE-7684)

### CODE REVIEW

**Comment by tkreuzer** [comment]
Awesome! We're becoming a modern OS, implementing even the most useless features from Windows ;-)

**Comment by sanchaez** [comment]
@tkreuzer it is not useless - it will be used for RAPPS! 😃

File: modules/rostests/apitests/com/com_apitest.h [hunk 1]
```diff
@@ -200,4 +200,12 @@ DEFINE_GUID(IID_CDefView,                  0x4434ff80, 0xef4c, 0x11ce, 0xae, 0x6
 DEFINE_GUID(IID_FolderItems2,              0xc94f0ad0, 0xf363, 0x11d2, 0xa3, 0x27, 0x00, 0xc0, 0x4f, 0x8e, 0xec, 0x7f);
 DEFINE_GUID(IID_FolderItems3,              0xeaa7c309, 0xbbec, 0x49d5, 0x82, 0x1d, 0x64, 0xd9, 0x66, 0xcb, 0x66, 0x7f);
 
+// zipfldr.dll
+DEFINE_GUID(CLSID_ZipFolderStorageHandler, 0xe88dcce0, 0xb7b3, 0x11d1, 0xa9, 0xf0, 0x00, 0xaa, 0x00, 0x60, 0xfa, 0x31);
```
Comment (ThFabba) [inline modules/rostests/apitests/com/com_apitest.h]: If you're going to add these to some shell-undoc header file later, you might as well do that now. Copying the GUIDs to com_apitest shouldn't become the norm :p

Comment (learn-more) [reply modules/rostests/apitests/com/com_apitest.h]: @tkreuzer Yeah, one of the first things I usually do in windows is install winrar, but it's pretty annoying when you are on a fresh ROS install and you can't open a .zip file ;)

@ThFabba I was not planning on doing that, I wanted to keep the GUID inside [shellext/zipfldr](https://github.com/learn-more/reactos/blob/zipfldr/dll/shellext/zipfldr/zipfldr.cpp)

**Pull request merged**